### DISCUSSION TITLE
AddFriend BotA BotB

### DISCUSSION THREAD

**Discussion by evgewakan** [chr233/ASFEnhance#44] [功能建议 / Ideas]
hello, is it possible to add a team to add as friends? 

AddFriend BotA BotB

*BotA send, BotB accept it request

**Comment by chr233** [comment]
it seems easy to implement, but I'm afraid I can't tell you when I will add this feature, it's a busy time.

**Comment by chr233** [comment]
Added, use 1.7.2.0 or later version.
